release notes/v0.55.0.md

```diff
@@ -0,0 +1,205 @@
+
+To overcome this issue in the browser module, we have implemented `page.on('metric')` which allows you to define url patterns using regex to match against. When a match is found the `url` and `name` tags for the metric will be replaced by the new `name`.
+
+#### Example usage
+
+```js
+
+import { browser } from 'k6/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+}
+
+export default async function() {
+  const page = await browser.newPage();
+
+  // Here we call page.on('metric). It's possible to call page.on('metric') more
+  // than once, and the callback function will be executed in the order page.on
+  // was called.
+  page.on('metric', (metric) => {
+    // At the moment metric.tag is the only method on the metric object. It is
+    // the method that will match on the url tag with the defined regex pattern.
+    // It can be called multiple times from within the callback function.
+    metric.tag({
+      // This is the new name that will be used for any metric which matches on
+      // the existing url tag.
+      name: 'test',
+      // You can provide multiple matches here. Any metrics that match will all
+      // use the new name defined above.
+      matches: [
+        // When a method is defined it will also need to match on that too. If a
+        // method is not provided it will match on all method types.
+        {url: /^https:\/\/test\.k6\.io\/\?q=[0-9a-z]+$/, method: 'GET'},
+      ]
+    });
+  });
+
+  try {
+    // This is only for illustrative purposes, the q query param doesn't affect
+    // the website.
+    await page.goto('https://test.k6.io/?q=abc123');
+    await page.goto('https://test.k6.io/?q=def456');
+  } finally {
+    await page.close();
+  }
+}
+
+```
+
+### `ControlOrMeta` support in the keyboard [browser#1457](https://github.com/grafana/xk6-browser/pull/1457)
+
+This will allow us to write the test for all platforms that either work with `Control` or `Meta` when performing keyboard actions. For example `control+click` on windows, and `meta+click` on mac to open a link in a new window.
+
+#### Example usage
+
+<details>
+<summary>Expand to see an example usage of `ControlOrMeta`</summary>
+
+```js
+  await page.keyboard.down('ControlOrMeta');
+
+  // Open the link in a new tab.
+  // Wait for the new page to be created.
+  const browserContext = browser.context();
+  const [newTab] = await Promise.all([
+    browserContext.waitForEvent('page'),
+    await page.locator('a[href="/my_messages.php"]').click()
+  ]);
+
+  await page.keyboard.up('ControlOrMeta');
+```
+
+</details>
+
+## UX improvements and enhancements
+
+- [browser#1462](https://github.com/grafana/xk6-browser/pull/1462) Enhance `waitForSelector` error message to better reflect why a selector doesn't resolve to an element.
```

Comment:
Here (and some other items below) needs the style alignment like we have in our release notes template:

```
_Format as `<number> <present_verb> <object>. <credit>`_:
```